### ISSUE TITLE
Automate Translation to Urdu with This Project

### ISSUE THREAD

**Issue by emadehsan** [angular-pakistan/angular.io.urdu#2]
Hello @saman1234, Probably it would be helpful if you automate the Urdu localization. You can utilize this project to minimize effort: https://github.com/emadehsan/ieighteen

**Comment by afirdousi** [comment]
Thanks for your suggestion @emadehsan. We are restarting this project and can definitely look into it. The challenge with this one is we are actually `not` looking for word-to-word translation. Also, we are not considering the translation to happen in old-school-pure-Urdu honestly. We are looking to translate the whole project into contemporary modern-day spoken Urdu.

Can you or your team volunteer/help us with this project?

**Comment by emadehsan** [comment]
Good to know. We can prolly contribute from time to time.

**Comment by afirdousi** [comment]
@emadehsan is it possible that we do one complete translation cycle with `ieighteen` and then translate the resultant into modern day urdu. what do you suggest?

**Comment by emadehsan** [comment]
Yup. And as `ieighteen` uses Google APIs, which are still not that good at Urdu translation, it's suggested to manually go through the translated text to proof read (and hence convert to modern day Urdu where needed).

**Comment by afirdousi** [comment]
@smkamranqadri  any suggestion?

**Comment by smkamranqadri** [comment]
Let me check ieighteen and then get back to you.
